docs/usage/private-modules.md

```diff
@@ -130,7 +130,7 @@ Any `hostRules` with `hostType=packagist` are also included.
 ### gomod
 
 If a `github.com` token is found in `hostRules`, then it is written out to local git config prior to running `go` commands.
-The command run is `git config --global url."https://${token}@github.com/".insteadOf "https://github.com/"`.
+???CHECK THIS LINE, IS THE COMMAND CORRECT???: The command run is `git config --global url."https://${token}@github.com/".insteadOf "https://github.com/"`.
```

Comment:
```suggestion
The command run is `git config --global url."https://${token}@github.com/".insteadOf"https://github.com/"`.
```

Is that whitespace between `.insteadOf "https://` correct or should it be removed?